circle.yml

```diff
@@ -13,6 +13,8 @@ machine:
         ANDROID_HOME: /usr/local/android-sdk-linux
         JAVA8_HOME:   /usr/lib/jvm/java-8-oracle
         JAVA7_HOME:   /usr/lib/jvm/java-7-openjdk-amd64
+        GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError"'
```

Comment:
Does it really work :heart_eyes: ??? can we have my SQL tests back :heart_eyes::heart_eyes::heart_eyes:???